haystack/nodes/answer_generator/openai.py

```diff
@@ -52,22 +53,24 @@ class OpenAIAnswerGenerator(BaseGenerator):
     def __init__(
         self,
         api_key: str,
-        model: str = "text-curie-001",
+        model: str = "text-davinci-003",
         max_tokens: int = 13,
         top_k: int = 5,
         temperature: float = 0.2,
-        presence_penalty: float = -2.0,
-        frequency_penalty: float = -2.0,
+        presence_penalty: float = 0.1,
+        frequency_penalty: float = 0.1,
         examples_context: Optional[str] = None,
-        examples: Optional[List] = None,
-        stop_words: Optional[List] = None,
+        examples: Optional[List[List[str]]] = None,
+        stop_words: Optional[List[str]] = None,
         progress_bar: bool = True,
+        instruction_prompt: Optional[PromptTemplate] = None,
```

Comment:
let's rename this one to `prompt_template` so that we are consistent with the naming in `PromptNode.prompt()`